actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -7,7 +7,7 @@ class Configuration
       attr_accessor :connection_class, :worker_pool_size
       attr_accessor :channel_load_paths
       attr_accessor :disable_request_forgery_protection, :allowed_request_origins
-      attr_accessor :cable, :url
```

Comment:
If we're keeping `action_cable_meta_tag`, doesn't it still use `url`?

I think we should keep `url`, but default it to `nil`, and then have `action_cable_meta_tag` use `url || mount_path || raise`.